fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -539,6 +541,40 @@ struct TeamCollectionInterface {
 	PromiseStream<GetTeamRequest> getTeam;
 };
 
+// Used to track the number of ongoing bulkload tasks for each storage server
+struct DDBulkLoadTaskBusyMap {
+public:
+	void addTask(const UID& ssid) {
+		auto it = busyMap.find(ssid);
+		if (it == busyMap.end()) {
+			busyMap[ssid] = 1;
+		} else {
+			it->second++;
+		}
```

Comment:
Nit: can be simplified as `busyMap[ssid]++;`, avoiding an extra lookup if not present. If not present, the entry will be created with `0`.